components/frontend-react/Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM node:lts
```

Comment:
Let's not go to bleeding edge. You can use specific version of node to be on safe side.